solutions/bio1/DebtRepayment/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { SCodeBlock, SText, STitle } from "components";
+
+export const DebtRepayment = () => {
+  return (
+    <>
+      <STitle>Idea</STitle>
+      <SText>
+        Due to the nature of the problem (at least $1%$ decreased each iteration), we can simply simulate the process until the debt reaches $0$.
+      </SText>
+      <SText>
+        Each iteration, we can calculate the amount to be repaid, and keep track of the total repaid amount in a variable.
+      </SText>
+
+      <STitle>Solution</STitle>
+      <SText>
+        To preprocess the inputs, we can first convert the integer percentages into decimal form (i.e. $10$ into $0.1$). This is done by dividing by $100$. Additionally, we should add $1$ to the interest amount, to allow easier multiplication when calculating the new debt.<br />
+        To avoid floating point math, we should keep track of the debt in cents. This way, we can avoid rounding errors. <br /><br />
```

Comment:
I don't like using `<br />`. If you want to begin a new paragraph, close the current `SText` and start a new one. That way, the formatting is consistent. I also don't recommend having a paragraph be a single line. Please either append it to the previous one or prepend it to the next one.